src/main/scala/clump/Clump.scala

```diff
@@ -67,15 +67,6 @@ object Clump {
 
   def collect[T](clumps: List[Clump[T]]): Clump[List[T]] =
     new ClumpCollect(clumps)
-
-  def sourceFrom[T, U](fetch: Set[T] => Future[Map[T, U]], maxBatchSize: Int = Int.MaxValue) =
```

Comment:
The new signatures and function transformations are so complex that I decided to move everything to ```ClumpSource``` and let the user use the object directly. I know that ideally we should hide the ```ClumpSource```, but i believe this is a good reason and Stitch also has something similar (```SeqGroup```)